services/plugins/generic_plugins/main.go

```diff
@@ -0,0 +1,17 @@
+package generic_plugins
+
+import (
+	"fmt"
+
```

Comment:
line

Reply:
no need as it's done like this in other files